### ISSUE TITLE
Update the node view after adding a widget

### ISSUE THREAD

**Issue by aloytag** [jchanvfx/NodeGraphQt#289]
When adding a widget to a node (e.g., a text input) I get this result:

![1](https://user-images.githubusercontent.com/36411739/209680149-b05217ce-db21-430c-a386-20c3a73c8d31.png)

...instead of this:

![2](https://user-images.githubusercontent.com/36411739/209680188-46a86621-42ea-4ce9-89a0-4ea054f77d7f.png)

I can get the look of the second image after changing something, like the value in the text input widget, or after executing the `update()` node method.

So I suggest adding `self.update()` as the last line in these four methods in `BaseNode` class (**base_node.py** file):
- `add_custom_widget()`
- `add_combo_menu()`
- `add_text_input()`
- `add_checkbox()`

Basically, adding `self.update()` after each `self.view.add_widget(widget)` execution.

**NodeGraphQt version:** 0.5.0
**Qt.py version:** 1.3.7
**PySide2 version:** 5.15.2.1
**Python version:** 3.9.2
**OS:** Debian GNU/Linux 11 (bullseye)

Regards.

**Comment by jchanvfx** [comment]
Cool thanks for catching that. I didn't take account that node widgets are added outside the `__init__` function.
